### ISSUE TITLE
manage ez notifications

### ISSUE THREAD

**Issue by dougplant** [mugoweb/eep#2]
list notifications with some interesting information
set and clear notifications

**Comment by benkmugo** [comment]
I've got list, create, remove for subtree notifications ready locally.
A few questions on that though.

1. I currently have the functions as part of the user module, since those types of notifications are user centric (and we already had that module). Do we want that or do we want a notifications module instead?

2. The list output currently looks like this. Showing the notification rule object fields as well as some extended node information. Thoughts?

```
 +-----+-------------+------+----------------------------------+------------+--------+
 I Notifications for user: 287 count: 1                                              |
 +-----+-------------+------+----------------------------------+------------+--------+
 I  Id | Use digest? | Node |                        Remote Id | Identifier |   Name |
 +-----+-------------+------+----------------------------------+------------+--------+
 | 267 |           0 |  269 | af8ca2e5b2a2c97ecf8a912105fc455c |     folder | Events |
 +-----+-------------+------+----------------------------------+------------+--------+
```

**Comment by benkmugo** [comment]
Addressed by PR #44
